engine/schema/src/main/resources/META-INF/db/schema-41800to41810.sql

```diff
@@ -47,3 +47,17 @@ WHERE so.default_use = 1 AND so.vm_type IN ('domainrouter', 'secondarystoragevm'
 -- fix erronous commas in guest_os names
 UPDATE `cloud`.`guest_os_hypervisor` SET guest_os_name = 'rhel9_64Guest' WHERE guest_os_name = 'rhel9_64Guest,';
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.guest_os', 'display', 'tinyint(1) DEFAULT ''1'' COMMENT ''should this guest_os be shown to the end user'' ');
+
+CREATE TABLE IF NOT EXISTS `cloud_usage`.`usage_vpc` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `vpc_id` bigint(20) unsigned NOT NULL,
+  `zone_id` bigint(20) unsigned NOT NULL,
+  `account_id` bigint(20) unsigned NOT NULL,
+  `domain_id` bigint(20) unsigned NOT NULL,
+  `state` varchar(100) DEFAULT NULL,
+  `created` datetime NOT NULL,
+  `removed` datetime DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB CHARSET=utf8;
+
+ALTER TABLE `cloud_usage`.`cloud_usage` ADD state VARCHAR(100) DEFAULT NULL;
```

Comment:
needs to be moved to a newer upgrade file

Reply:
@SadiJr this really nees addressing.

Reply:
@DaanHoogland sorry for the delay, and done, thanks for your review.